Specs/Core/VideoSynchronizerSpec.js

```diff
@@ -41,6 +41,26 @@ defineSuite([
         return element;
     }
 
+    function MockVideoElement() {
+        this.paused = false;
```

Comment:
We prefer to use jasmine spies rather than create helper objects like this.  Spies are more flexible and allow you to inspect what methods are actually getting called instead of managing state yourself.  For example, you could replace this whole class with:

```js
var element = jasmine.createSpyObj('MockVideoElement', ['play', 'pause', 'addEventListener', 'removeEventListener']);
```

And then instead of checking a `paused` variable you would make calls like: `expect(element.play).toHaveBeenCalled()`. after you expect play to have been called (same with pause).